packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.d.ts

```diff
@@ -4,8 +4,11 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { Constructor } from '@open-wc/dedupe-mixin';
+import { ResizableMixinClass } from '@vaadin/component-base/src/resizable-mixin';
```

Comment:
nit: let's add `.js` extension for consistency.

Reply:
Done